client/client.go

```diff
@@ -29,49 +31,52 @@ type HttpFunc func(ctx context.Context, url goja.Value, args ...goja.Value) (*k6
 
 func New(vu modules.VU) *TracingClient {
 	return &TracingClient{
-		http: &k6HTTP.HTTP{},
+		http: &k6HTTP.Client{},
```

Comment:
That will just not work as later on you need it's internal fields to be useful see #21 for my at least working fixes ... although arguably the code is a lot worse ;(

Reply:
True :disappointed: 

I'll check if I can come up with something else, but for now I belive maybe better to merge your PR :+1: